### DISCUSSION TITLE
RSS feeds for new Apps on Github

### DISCUSSION THREAD

**Discussion by jbionic2016** [community/community#22773] [New to GitHub]
<p>Hi there,<br>
I’ve just read a few most recent topics with word RSS in their title, but still somehow miss to understand what is the best way to subscribe to Github’s newsline that will highlight new apps that have been shared through GitHub. As an amateur user I would specifically target apps that are free and verified.<br>
Thanks.</p>

**Comment by ernest-phillips** [comment]
<p>Hi <a class="mention" href="/jbionic2016">@jbionic2016</a>,</p>
<p>Thanks for coming to the community!</p>
<p>Could you explain more about the specific information you are trying to be notified about? What does “free and verified” mean? This may help the community recommend a service that matches your needs.</p>
<p>GitHub uses the Explore page to showcase projects that you may be interested in.<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/explore" target="_blank" rel="noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://user-images.githubusercontent.com/5134583/181073098-82664c46-9be2-4892-999f-76ec85b102f6.jpeg" class="thumbnail" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/7/97a61954e264355727123e49e128f43cc7391f6c_2_690x362.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/9/7/97a61954e264355727123e49e128f43cc7391f6c_2_1035x543.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/7/97a61954e264355727123e49e128f43cc7391f6c.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/7/97a61954e264355727123e49e128f43cc7391f6c_2_10x10.png"></div>

<h3><a href="https://github.com/explore" target="_blank" rel="noopener">Explore GitHub</a></h3>

<p>Explore is your guide to finding your next project, catching up with what’s trending, and connecting with the GitHub community.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Reply by jbionic2016** [reply]
Thanks. It's been nearly 4 years with no notion of any movement on GitHub's side, so I did as you advised and duplicated my question in the following discussion just opened 
https://github.com/orgs/community/discussions/61463

Hopefully, in 10 years we'll see a response...

**Comment by jbionic2016** [comment]
<p>Hi Ernst,</p>
<p>Thanks for coming back to me to clarify my request.</p>
<p>GitHub is a repository of apps. Every week new apps are added to GitHub and older apps are updated by developers. According to the below article, some apps are tagged as verified, the rest are unverified:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://developer.github.com/marketplace/" target="_blank" rel="noopener nofollow ugc">GitHub Developer</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/23212486/181073099-58a251e8-95bc-433d-a373-0ec7637b2b27.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_10x10.png">

<h3><a href="https://developer.github.com/marketplace/" target="_blank" rel="noopener nofollow ugc">GitHub Marketplace</a></h3>

<p>Get started with one of our guides, or jump straight into the API documentation.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Ideally, I’d like to subscribe to GitHub’s RSS/Atom feed in order to get weekly notifications of the new apps added to GitHub. Since all apps can be grouped by topic, I’d love my subscriptions to be adjustable by topic name and status of their verification.</p>
<p>Hope this helps</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi :wave:</p>
<p>Thanks for clarifying the use case here. We do not have such a service feature for the Marketplace.<br>
However, we’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.<br>
Would you mind <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request?</p>
